src/EnergyPlus/FluidProperties.hh

```diff
@@ -749,9 +749,6 @@ struct FluidPropertiesData : BaseGlobalStruct
     int RefrigerantErrorLimitTest = 1; // how many times error is printed with details before recurring called
     Array1D_bool RefrigUsed;
     Array1D_bool GlycolUsed;
-    int FluidIndex_Water = 0;
-    int FluidIndex_EthyleneGlycol = 0;
-    int FluidIndex_PropoleneGlycol = 0;
```

Comment:
These global vars were removed because they were not used anywhere. There was a minor modification to ```EnergyPlusFixture``` to replicate the effects of calling ```FindGlycol``` for each of the fluids.

Reply:
More removed from state.  Nice.